Misc/NEWS.d/next/Library/2022-06-03-22-13-28.gh-issue-93370.tjfu9L.rst

```diff
@@ -0,0 +1 @@
+deprecate sqlite3.version and sqlite3.version_info.
```

Comment:
Not sure if ref is correct here as the attributes are documented with `:data:` and I don't have a local checkout right now to test.
```suggestion
Deprecate :ref:`sqlite3.version` and :ref:`sqlite3.version_info`.
```

https://github.com/python/cpython/blob/7b3cf3a2884b130c519fbe271e4d2a762248c2a0/Doc/library/sqlite3.rst?plain=1#L140-L149

Reply:
Looks like we need to use `:data:` - [:py:data:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-data)